### DISCUSSION TITLE
Available feature "Language recognition" in other language than Ruby

### DISCUSSION THREAD

**Discussion by brunolnetto** [github-linguist/linguist#6207] [General]
## Describe 

"Languages are not equal. Therefore, should they deserve more than others?" 

With this Orwell-like question I introduce my post on language detection in a context broader than language literation, related to code comprehension and execution. It stems from the inconvenient truth that some feature or another is available in one language and not in another.

Therefore, I ask myself: is there for this specific feature, language recognition, some available alternative in another language?

**Comment by lildude** [comment]
Wow! If I'm understanding you correctly, this is a very long winded way of asking is there a equivalent to Linguist written in another language that isn't Ruby. Yes there is: [go-enry](https://github.com/go-enry/go-enry), which feeds directly off Linguist, is written in Go. GitHub uses go-enry as part of its language detection in its search functionality.

This isn't really an issue so I'm transferring this to discussions.

**Reply by brunolnetto** [reply]
Long-winded comments are bad received by community. Weird.
